IPMUinternal/PFS-IPMU-00001/maintenance.rst

```diff
@@ -44,6 +44,24 @@ use one at `/server/admin/github-scripts/pfs_infra_ipmu/ldap-admin/`.
     default gid is from opt_gid (2000)
     fullname is an option, from third to the end when defined
 
+Here is an example to create a new account:
+
+::
+
+  root@pfs:~# cd /server/admin/ldif
+  root@pfs:/server/admin/ldif# cat > yyyymmdd
+  <user e-mail address> <username> <Full Name>
+  ^D
+    # If there is a picture to upload, place username.jpg in the same directory to run the script
+  root@pfs:/server/admin/ldif# ./create-ldif.pl --jira yyyymmdd
+    # if the user requires access to JIRA, use --jira option
+    # replace --jira by --reset if you want to reset password
```

Comment:
This sentence is confusing.
I think you should make another example to "reset" an account.